include/nuttx/spi/qspi.h

```diff
@@ -271,6 +319,10 @@ struct qspi_ops_s
   CODE void      (*setmode)(FAR struct qspi_dev_s *dev,
                     enum qspi_mode_e mode);
   CODE void      (*setbits)(FAR struct qspi_dev_s *dev, int nbits);
+#ifdef CONFIG_QSPI_HWFEATURES
+  CODE int       (*hwfeatures)(FAR struct qspi_dev_s *dev,
+                    qspi_hwfeatures_t features);
+#endif
```

Comment:
Should we update all `struct qspi_ops_s` instances to add
```
#ifdef CONFIG_QSPI_HWFEATURES
  , NULL
#endif
```
? Otherwise existing drivers will be broken

Reply:
another method is moved the new callback to the end of struct